Dockerfile

```diff
@@ -8,8 +8,8 @@ WORKDIR /app
 
 # Install dependencies and remove apt cache
 RUN apt-get update && apt-get install -y \
+  \
```

Comment:
not familiar with DockerFile notation, but does dit backslash don't do anyhting? Then better just not touch the file.

Reply:
I've removed it.